Firestore/Swift/Source/Codable/ExplicitNull.swift

```diff
@@ -39,7 +39,7 @@ public enum ExplicitNull<Wrapped> {
   }
 
   /// Get the `Optional` representation of `ExplicitNull`.
```

Comment:
Comments should be phrased with an implicit "This method ..."

What do you think of rephasing this as

> Returns this value as an `Optional<Wrapped>`?

Reply:
Better, done.